pages/about.html

```diff
@@ -1,12 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/about_styles.css">
 </head>
+
 <body>
-  
+    <section class="container">
```

Comment:
This doesn't really have semantic value. Consider dropping the outer section (which would also fix the `main` validation error) so that `header`, `main`, and `footer` are all direct children of `body`.